packages/xod-cli/README.md

```diff
@@ -9,23 +9,266 @@ other packages. The responsibility of `xod-cli` is to parse command line
 arguments, call API, and format the result on stdout/stderr properly.
 
 `xodc` uses subcommands like `git` does to perform various functions.
-The subcommands handling could be found in `src/xodc-*.js`.
-
-## Usage
-
-```
-  xodc pack <projectDir> <output>
-  xodc unpack <xodball> <workspace>
-  xodc transpile [--output=<filename>] [--target=<target>] <input> <path>
-  xodc publish --swagger=<swagger> --author=<author> [--orgname=<orgname>] [<projectDir>]
-  xodc install --swagger=<swagger> <libUri> [<path>]
-  xodc ab set-executable <path>
-  xodc ab set-packages <path>
-  xodc ab list-index
-  xodc ab list-pavs
-  xodc ab list-ports
-  xodc ab list-boards <package> <architecture> <version>
-  xodc ab install-pav <package> <architecture> <version>
-  xodc ab compile <package> <architecture> <board> <file>
-  xodc ab upload <package> <architecture> <board> <port> <file>
+The subcommands handling could be found in `src/commands/*.js`.
+
+![xod-cli](https://img.shields.io/badge/xod-cli-brightgreen.svg)
+[![Version](https://img.shields.io/npm/v/xod-cli.svg)](https://npmjs.org/package/xod-cli)
+[![Downloads/week](https://img.shields.io/npm/dw/xod-cli.svg)](https://npmjs.org/package/xod-cli)
+[![License](https://img.shields.io/npm/l/xod-cli.svg)](https://github.com/xodio/xod/blob/master/packages/xod-cli/package.json)
+
+<!-- toc -->
+* [xod-cli](#xod-cli)
+* [Usage](#usage)
+* [Flags, aliases, environment variables](#flags-aliases-environment-variables)
+* [Commands](#commands)
+<!-- tocstop -->
+# Usage
+<!-- usage -->
+```sh-session
+$ npm install -g xod-cli
+$ xodc COMMAND
+running command...
+$ xodc (-v|--version|version)
+xod-cli/0.25.0 linux-x64 node-v10.11.0
+$ xodc --help [COMMAND]
+USAGE
+  $ xodc COMMAND
+...
+```
+<!-- usagestop -->
+
+# Flags, aliases, environment variables
+
+Almost any flag can be replaced with the appropriate environment variable.
+For example, instead of `--username` you can declare variable `XOD_USERNAME`.
+
+| Flag         | Alias | Environment variable |
+|--------------|-------|----------------------|
+| --api        |       | XOD_API              |
+| --debug      |       | XOD_DEBUG            |
+| --help       | -h    |                      |
+| --no-build   |       |                      |
+| --on-behalf  |       | XOD_ONBEHALF         |
+| --output     | -o    | XOD_OUTPUT           |
+| --output-dir | -o    | XOD_OUTPUT           |
+| --password   | -p    | XOD_PASSWORD         |
+| --quiet      | -g    |                      |
```

Comment:
Wrong alias defined

Reply:
`output` vs `output-dir`? They are from different commands.

Reply:
-g? Ok, I correct this.